### ISSUE TITLE
Hey mate

### ISSUE THREAD

**Issue by blinkinglight** [mr-karan/coredns-nomad#1]
i will try to move github actions today, 
also mine has failover and it accepts few nomad servers to try if one dies, so maybe later we could look how to merge that too? what do you think, because restart DNS with few hundreds services to change nomad master for me looks not as good approarch

so, what do you think? :)

**Comment by mr-karan** [comment]
Yeah, we can add support for multiple Nomad servers. Ideally, in my environment I have a proxy which is frontending multiple servers in background, so the LB takes care of failovers across multiple servers. But if it doesn’t make the code too complicated, we can do multiple servers. 

The GH Actions part also can be added here, a separate PR for that is better.

**Comment by blinkinglight** [comment]
GH actions is very very simple
